tests/CodeSnifferRunner.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace ObjectCalisthenics\Tests;
+
+use PHP_CodeSniffer;
+
+final class CodeSnifferRunner
+{
+    /**
+     * @var PHP_CodeSniffer
+     */
+    private $codeSniffer;
+
+    public function __construct()
+    {
+        $this->codeSniffer = new PHP_CodeSniffer();
+    }
+
+    /**
+     * @param string $testedFile
+     * @param string $sniffName
+     *
+     * @return int
+     */
+    public function detectErrorCountInFileForSniff($testedFile, $sniffName)
+    {
+        $this->codeSniffer->initStandard('../../../../../src/ObjectCalisthenics', [$sniffName]);
```

Comment:
I'm glad it's back.

To this one: https://github.com/object-calisthenics/phpcs-calisthenics-rules/blob/master/ruleset.xml

Could you try if your solution works?